webapp/sources/api-doc/paths/directives/all.yml

```diff
@@ -42,50 +42,3 @@ get:
     - lang: curl
       source:
         $ref: ../../code_samples/curl/directives/all.sh
-put:
```

Comment:
why removing this?

Reply:
certainly because it remove it in the incorrect file (it was the source from witch I copy/pasted the structure. Tabs in editor are complicated).